searx/engines/tagesschau.py

```diff
@@ -83,7 +83,8 @@ def _story(item):
 
 
 def _video(item):
-    video_url = item['streams']['h264s']
+    streams = item['streams']
+    video_url = streams.get('h264s') or streams.get('h264m') or streams.get('h264l') or streams.get('h264xl')
```

Comment:
No need for, just to your knowledge: this can also be written by:

```python
streams.get('h264s', streams.get('h264m', streams.get('h264l', streams.get('h264xl'))))
```